region/maridia/outer.json

```diff
@@ -2423,6 +2485,40 @@
                   "devNote": "FIXME: It is possible to get over the space pirate, damageless, but it is challenging, especially when keeping the Skulltera alive for other frozen strats."
                 }
               ]
+            },
+            {
+              "id": 7,
+              "strats": [
+                {
+                  "name": "G-Mode Morph IBJ",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    "Gravity",
+                    {"or": [
+                      {"and": [
+                        "h_canArtificialMorphIBJ",
+                        "canTrickyJump",
+                        "canBombHorizontally"
+                      ]},
+                      {"and": [
+                        "h_canArtificialMorphSpringBallBombJump",
+                        "h_canArtificialMorphSpringBallBombJump"
```

Comment:
I see this as 3 springballbombjumps.
Or 1 sbbj and landing near two pirates.  But standing near pirates wouldn't need GMode.

Reply:
I didnt even think of landing on the platform with the last pirate. This is just a careful jump, which I have been counting as free in G-Mode

Reply:
This whole strat is not useful either, as it can be done with gravity alone